spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/CMulSpec.scala

```diff
@@ -65,6 +66,22 @@ class CMulSpec extends FlatSpec with Matchers {
     gradInput3 should be (gradInput4)
     layer2.gradWeight should be (layer1.gradWeight.mul(0.5))
   }
+
+  "CMUl" should "work good on batch input" in {
```

Comment:
it's for special batch size right ?

Reply:
yes, e.g.  batchsize equals channel number, then there will be wrong result.